reledmac.dtx

```diff
@@ -12070,18 +12268,23 @@
 % \item \protect\cs{ifl@d@esl} for ending sub-line; and
 % \item \protect\cs{ifl@d@dash} for the dash between the starting and ending groups.
 % \end{itemize}
-% There is no boolean for the line number because it is always printed.
+% There was no boolean for the line number because it was always printed.
+%
+% Maïeul Rouquette has added:
+% \begin{itemize}
+% \item \protect\cs{ifl@d@Xtwolines} and \protect\cs{ifl@d@Xmorethantwolines}  to print a symbol which stands for ``and subsequent`` when there are two, three or more lines;
```

Comment:
"``and subsequent``" → "``and subsequent''"

Reply:
indeed, but I should use csquotes package (which I loaded !)